nautobot_floor_plan/forms.py

```diff
@@ -37,6 +37,18 @@ class FloorPlanForm(NautobotModelForm):
         help_text="The first value to begin Y Axis at.",
         required=False,
     )
+    x_axis_step = forms.IntegerField(
+        label="X Axis Step",
+        help_text="A positive or negative integer, excluding zero",
+        required=False,
+        validators=[utils.validate_not_zero],
+    )
+    y_axis_step = forms.IntegerField(
+        label="Y Axis Step",
+        help_text="A positive or negative integer, excluding zero",
+        required=False,
+        validators=[utils.validate_not_zero],
+    )
```

Comment:
Edge case: using A/-1 causes blank where Z should be:
![Screenshot 2024-08-16 at 9 21 32 AM](https://github.com/user-attachments/assets/8dbbaa92-a5b2-4c6a-99a3-e0afcaa05f48)

Reply:
Add logic not use the number 0 when letters are used for labeling.